src/main/java/de/retest/web/DefaultValuesProvider.java

```diff
@@ -0,0 +1,82 @@
+package de.retest.web;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+import java.util.Map.Entry;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+
+import de.retest.ui.DefaultValueFinder;
+import de.retest.ui.descriptors.IdentifyingAttributes;
+
+public class DefaultValuesProvider implements DefaultValueFinder {
+
+	public static final String DEFAULTS_FILE_PATH = "/defaults.yaml";
+
+	private final Map<String, Map<String, String>> defaultValues;
+
+	public DefaultValuesProvider() {
+		try ( final InputStream url = getClass().getResourceAsStream( DEFAULTS_FILE_PATH ) ) {
+			defaultValues = readAttributesConfigFromFile( url );
+		} catch ( final Exception e ) {
+			// TODO Own exception
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://sonarcloud.io/organizations/retest/rules#rule_key=squid%3AS1135))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=de.retest%3Arecheck-web&pullRequest=50&issues=AWUPD_F88anAXINBxwKX&open=AWUPD_F88anAXINBxwKX)

[ISSUE_KEY]: AWUPD_F88anAXINBxwKX

Reply:
Done